react/features/file-sharing/components/web/FileSharing.tsx

```diff
@@ -283,98 +280,103 @@ const FileSharing = () => {
                                 ref = { fileInputRef }
                                 type = 'file' />
                         </div>
-                        { files.size === 0 && (
-                            <div className = { classes.noFilesContainer }>
-                                <Icon
-                                    className = { classes.uploadIcon }
-                                    color = { BaseTheme.palette.icon03 }
-                                    size = { 160 }
-                                    src = { IconCloudUpload } />
-                                <span className = { classes.noFilesText }>
-                                    { t('fileSharing.dragAndDrop') }
-                                </span>
-                            </div>
-                        ) }
+                        {
+                            files.size === 0 && (
```

Comment:
`files` is now a sorted array, so you need `.length` here don't you?

Reply:
If we use the newly created variable sortedFiles, we do, yes. I will replace these as well.